site/en/docs/privacy-sandbox/faq/index.md

```diff
@@ -185,6 +185,12 @@ affect other pages.
 
 Topics are only inferred from the hostname and not from the URL path.
 
+### Can I control which third parties on my page are allowed to infer topics from my users?
```

Comment:
```suggestion
### Can I control which third parties have access to topics on my page?
```

Reply:
@antoinebisch I want to confirm this doesn't change the meaning. Otherwise looks good